<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background: rgb(67, 83, 106);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background: rgb(67, 83, 106);</string>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="startPage">
       <property name="styleSheet">
        <string notr="true">background: rgb(67, 83, 106)</string>
       </property>
       <widget class="QHoverSensitiveButton" name="fetchButton">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>30</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(63, 76, 255);
border-style: solid;

</string>
        </property>
        <property name="text">
         <string>Fetch</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="settingButton">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>160</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 233, 39);
border-style: solid;
max-width:100px;
max-height:100px;
min-width:100px;
min-height:100px;
</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Narrow</family>
          <pointsize>15</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Hover + Hold</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Narrow</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>to Select</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="tutorialButton">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(90, 225, 143);
border-style: solid;
max-width:100px;
max-height:100px;
min-width:100px;
min-height:100px;
</string>
        </property>
        <property name="text">
         <string>Tutorials</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="movementPage">
       <widget class="QHoverSensitiveButton" name="releaseButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>170</y>
          <width>122</width>
          <height>102</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(90, 225, 143);
border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: rgb(18, 121, 65);
 max-width:120px;
 max-height:100px;
 min-width:120px;
 min-height:100px;</string>
        </property>
        <property name="text">
         <string>Release</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="backButton_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: red;
border: solid;</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="forwardButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>131</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 223, 51);
border: solid;</string>
        </property>
        <property name="text">
         <string>Forward</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="backwardButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>131</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(29, 241, 240);
border: solid;</string>
        </property>
        <property name="text">
         <string>Backward</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="upButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(205, 205, 205);
border: solid;</string>
        </property>
        <property name="text">
         <string>Up</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="leftButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(205, 205, 205);
border: solid;</string>
        </property>
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="rightButton">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>60</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(205, 205, 205);
border: solid;</string>
        </property>
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="downButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>110</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(205, 205, 205);
border: solid;</string>
        </property>
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <property name="styleSheet">
        <string notr="true">background: rgb(67, 83, 106)</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>30</y>
          <width>391</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-alignment: AlignCenter;</string>
        </property>
        <property name="text">
         <string>SETTINGS</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="backButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: red;
border: solid;</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="hoverButton">
        <property name="geometry">
         <rect>
          <x>31</x>
          <y>90</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>ON/OFF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>100</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-alignment: AlignCenter;</string>
        </property>
        <property name="text">
         <string>Hover Mode</string>
        </property>
       </widget>
       <widget class="QLabel" name="hover_time_label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>155</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-alignment: AlignCenter;</string>
        </property>
        <property name="text">
         <string>3 secs</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="hover_time_up_button">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>51</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 223, 51);
border: 1px solid black;
</string>
        </property>
        <property name="text">
         <string>U</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="hover_time_down_button">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>180</y>
          <width>51</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(127, 169, 255);
border: 1px solid black;</string>
        </property>
        <property name="text">
         <string>D</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-alignment: AlignCenter;</string>
        </property>
        <property name="text">
         <string>Hover Time</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tutorialPage">
       <widget class="QHoverSensitiveButton" name="backButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: red;
border: solid;</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>391</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-alignment: AlignCenter;</string>
        </property>
        <property name="text">
         <string>TUTORIALS</string>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>381</width>
          <height>211</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Controls</string>
         </attribute>
         <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>351</width>
            <height>181</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(107, 164, 227)</string>
          </property>
          <widget class="QHoverSensitiveButton" name="fetchButton_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>101</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(63, 76, 255);
border: 1px solid black;
font: 14pt ;
</string>
           </property>
           <property name="text">
            <string>Fetch</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>91</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt ;</string>
           </property>
           <property name="text">
            <string>Hover + Hold</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>40</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt;</string>
           </property>
           <property name="text">
            <string>to Select</string>
           </property>
          </widget>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>10</y>
             <width>191</width>
             <height>151</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(212, 212, 212)</string>
           </property>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>171</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Move cursor over buttons</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>171</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Hold for ~3 secs</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>80</y>
              <width>171</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Fetch will call the arm</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>100</y>
              <width>171</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>to fetch the wand</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QHoverSensitiveButton</class>
   <extends>QPushButton</extends>
   <header>qhoversensitivebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
